src/awscdk/awscdk-app-ts.ts

```diff
@@ -205,7 +205,8 @@ export class AwsCdkTypeScriptApp extends TypeScriptAppProject {
     }
 
     this.cdkConfig = new CdkConfig(this, {
-      featureFlags: this.cdkDeps.cdkMajorVersion < 2,
+      // featureFlags: this.cdkDeps.cdkMajorVersion < 2,
```

Comment:
Guess we should have a new top-level option to confiugure this?
```suggestion
      featureFlags: options.featureFlags ?? this.cdkDeps.cdkMajorVersion < 2,
```